Makefile.sp

```diff
@@ -33,7 +33,7 @@ $(SPHINXDIR)/requirements.txt:
 $(VENVDIR): $(SPHINXDIR)/requirements.txt
 	@echo "... setting up virtualenv"
 	python3 -m venv $(VENVDIR)
-	. $(VENV); pip install --require-virtualenv \
+	. $(VENV); pip install $(PIPOPTS) --require-virtualenv \
```

Comment:
I just merged a PR that adds a second `pip install` to this Makefile. I don't think we need the PIPOPTS there (because that command installs custom prerequisites that are empty by default), but you might want to confirm. Or we might want to add them for consistency?

Reply:
I'll add it both places for consistency, thank you for pointing it out.